src/components/select/select.component.spec.ts

```diff
@@ -229,18 +230,16 @@ describe('Select Component', () => {
         expect(component.value).toBe('One');
     });
 
-    it('should not open dropdown when tabbing past select', () => {
-        fixture.detectChanges();
-
-        const tabEvent: Event = new KeyboardEvent('keydown', {
-            'code': '9'
-        });
-        for (let i = 0; i < 5; i++) {
-            window.dispatchEvent(tabEvent);
+    fit('should not open dropdown when tabbing past select', (done) => {
+        for (let i = 0; i < 10; i++) {
+            dispatchKeyboardEvent(document, 'keydown', 9);
```

Comment:
Whenever you need to use keycodes, the `@angular/cdk` packages exports all the keycodes for us to use so we don't have to use hard coded numbers.